src/lib/y2partitioner/actions/controllers/fstabs.rb

```diff
@@ -127,6 +140,13 @@ def disk_analyzer
           DeviceGraphs.instance.disk_analyzer
         end
 
+        # Current system architecture
+        #
+        # @return [Storage::Arch]
+        def arch
+          @arch ||= Y2Storage::StorageManager.instance.arch
+        end
```

Comment:
Do we need something that `Yast::Arch` cannot provide? If not, I would use `Yast::Arch` (just like the rest of YaST and most places of y2-storage-ng). The StorageManager arch is only used in y2-storage-ng when `Arch#efiboot?` if needed. In all other places it uses `Yast::Arch`.